tests/ui/transmutability/visibility/should_reject_if_dst_has_private_field.stderr

```diff
@@ -2,7 +2,10 @@ error[E0277]: `Src` cannot be safely transmuted into `Dst` in the defining scope
   --> $DIR/should_reject_if_dst_has_private_field.rs:35:41
    |
 LL |     assert::is_transmutable::<src::Src, dst::Dst, Context>();
-   |                                         ^^^^^^^^ `Src` cannot be safely transmuted into `Dst` in the defining scope of `test::Context`.
+   |                                         ^^^^^^^^
+   |                                         |
+   |                                         `Src` cannot be safely transmuted into `Dst` in the defining scope of `test::Context`.
+   |                                         Reason: DstIsPrivate
```

Comment:
I'd suggest:
```
`Src` cannot be safely transmuted into `Dst` in the defining scope of `test::Context` because `Dst` is or contains a type or field that is not visible in that scope.
```